platform-sdk/docs/base/logging/logging.md

```diff
@@ -0,0 +1,210 @@
+## Hedera Logging API
+
+> [!WARNING]  
+> The API is currently in develpment and is not optimized for production use.
+ It is subject to change without notice.
+
+
+### Introduction
+The Hedera Logging API is a custom logging facade which is crafted to meet our unique requirements, ensuring compatibility with various software integrations.
+It acts as a bridge, redirecting events from eg SLF4J to our facade, providing flexibility for all dependencies used in the project.
+
+
+### Architecture and Configuration
+The architecture does not dictate the format of logging output, allowing for a wide range of output options.
```

Comment:
Before the mentioned points I would add:
as any project of Base the logging is splitted in a public (api) and private (implementation) part and provides extension points next to a very basic functionality. For the logging lib Logging handlers and provider are extension points.
Another goal of the architecture is to not reinvent the wheel how logging in Java works. If you familiar with any Java logging lib you should be more or less ready to use our logger api.